tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -97,4 +99,8 @@ def test_nodereplacement_proactive(self, pvc_factory, pod_factory, dc_pod_factor
         self.sanity_helpers.delete_resources()
         # Verify everything running fine
         log.info("Verifying All resources are Running and matches expected result")
-        self.sanity_helpers.health_check()
+        self.sanity_helpers.health_check(cluster_check=False)
```

Comment:
Here, you are just ignore the cluster health check. Still the ceph_health_check will happen and test case may fail if cluster is not healthy.